src/hooks/usePager.js

```diff
@@ -13,6 +13,7 @@ export default ({ numberOfPages }) => {
     pageSize,
     totalPages,
     setPage,
-    setPageSize
+    setPageSize,
+    hasNextPage
```

Comment:
hasNextPage is derived data, therefore ideally it would not be stored in the state but instead computed like totalPages and pages are above